ui/apps/platform/src/Containers/AppPage.tsx

```diff
@@ -19,7 +19,7 @@ function AppPage(): ReactElement {
                 <Route path={loginPath} component={LoginPage} />
                 <Route path={testLoginResultsPath} component={TestLoginResultsPage} />
                 <Route path={authResponsePrefix} component={LoadingSection} />
-                <Redirect from="/" to={mainPath} />
+                <Route path="/" exact render={() => <Redirect to={mainPath} />} />
```

Comment:
For my own understanding, what is the reason we need to have `exact` here for this route? I'm guessing that this is what leads to the case where "https://localhost:3000/unknown" renders a black screen. Does having a catch-all for routes outside of `/main/` that redirects to `/main/dashboard` conflict with what we are going for with the new 404 rendering?

Reply:
It appears that removing the `exact` will cause URLs like https://localhost:3000/unknown to redirect to https://localhost:3000/main/dashboard.

I'm okay with that for now, as opposed to the blank screen.

Reply:
Good catch. I will delete `exact` prop.

There is no reason to change previous behavior that /unknown redirects to /main and then to /main/dashboard.

Reply:
For the next iteration, we might as well consider why we have `/main` at all. 

@sachaudh @alwayshooin Do either of you know why there's a root URL segment "main", instead just using `/` ?

Reply:
I don't think there was a significance to adding it. If we decide to remove it, I think that's fine. I'm wondering what that means for folks that might have saved/bookmarked the URLs though. If it's nothing that would make it harder for customers, I think removing that would be fine.

Reply:
We don't have to remove it. We just have to know why it's there, and "because history" is a valid reason.